src/IntegrationUtils.ts

```diff
@@ -32,8 +32,8 @@ import { TelemetryEvent } from '@redhat-developer/vscode-redhat-telemetry/lib';
 const validNameRegex = /^[A-Za-z][A-Za-z0-9\-\.]*(?:[A-Za-z0-9]$){1}/;
 const devModeIntegration: string = 'Dev Mode - Apache Camel K Integration in Dev Mode';
 export const basicIntegration: string = 'Basic - Apache Camel K Integration without extra options';
-export const configMapIntegration: string = 'ConfigMap - Apache Camel K Integration with Kubernetes ConfigMap';
-export const secretIntegration: string = 'Secret - Apache Camel K Integration with Kubernetes Secret';
+export const configMapIntegration: string = 'ConfigMap - Apache Camel K Integration with Kubernetes ConfigMap as runtime configuration';
```

Comment:
Would adjust capitalization of the new text to match the previous text: "as Runtime Configuration"